### ISSUE TITLE
Option to squash to a file and not create an artifact

### ISSUE THREAD

**Issue by 2mal3** [ComunidadAylas/PackSquash-action#62]
### Related problem

The action always generates an artifact for the squashed datapack or resourcepack, but this is a bit inconvenient for some workflows. For example, if you want to modify the squashed pack in the same job, you have to download the artifact again after squashing to be able to use it, which makes the workflow longer and creates unnecessary arifacts.

### Proposed solution

A conceivable solution would be, for example, to specify the [option_file_path](https://github.com/ComunidadAylas/PackSquash/wiki/Options-files#output_file_path) which already exists in the normal packsuqash, as an option. If the path is not empty, it could be passed to packsquash, so the pack would be saved there instead of being uploaded as an artifact.
This would have the additional advantage that you could specify the name of the pack directly, without having to change it later.

### Alternative solutions

Of course, if the idea can't be implemented, you can just use the old artifact solution without any problems.

**Comment by AlexTMjugador** [comment]
Hi! Thank you for sharing your feedback :smiley: 

As you have hinted to in your issue, the option you described was never implemented because it's not strictly necessary to achieve anything, and the caching logic relies on previously uploaded artifacts to work.

However, I see some value in keeping workflows more concise for the likely common case of modifying the pack in the same run, and recognize that downloading its artifact may be too verbose. I think that your idea is worth looking into at some point.

By the way, third-party actions such as [`delete-artifact`](https://github.com/marketplace/actions/delete-artifact) can delete artifacts that were uploaded in a previous step, so that's a good way to deal with leaving unused artifacts behind, if that's your greatest concern. Of course, executing such actions comes at the price of even more verbosity.

**Comment by AlexTMjugador** [comment]
I've just pushed a commit that allows overriding the value of `output_file_path` in the PackSquash options, so that the action can instruct PackSquash to generate its ZIP file wherever necessary. If this option is not specified, the action will create the ZIP file in a temporary internal directory.

However, for the reasons given above, I've decided that generating artifacts is still a good idea. The extra verbosity of deleting artifacts is not much greater than writing an extra input to opt out of this generally beneficial behavior.

Do these changes sound good to you? :heart:
